src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -553,14 +553,17 @@ bool clang_c_convertert::get_var(const clang::VarDecl &vd, exprt &new_expr)
     if(r)
       return true;
 
+    bool aggregate_without_init = aggregate_value && val.id() == "sideeffect" &&
```

Comment:
Is there another way to check explicit initialisation in the clang AST `vd`?
We usually inspect clang AST to determine how we proceed with constructing the irept. 
Since `val` is our irept, relying on the characteristics of irept may not do good to generality.

Reply:
Maybe we can modify it in this method? Do you have any thoughts? I don't know exactly how this method is called.

https://github.com/esbmc/esbmc/blob/5f1c964abff300466a10ee5b049e98044a0cc5c1/src/clang-cpp-frontend/clang_cpp_main.cpp#L8-L16